zebra-chain/src/transaction/hash.rs

```diff
@@ -13,7 +13,7 @@ use super::Transaction;
 ///
 /// TODO: I'm pretty sure this is also a SHA256d hash but I haven't
 /// confirmed it yet.
```

Comment:
unrelated to this PR but just because it's surfaced here: we should check that the transaction hash logic is correct and then remove this comment

Reply:
#972